content/04.results.md

```diff
@@ -229,7 +227,7 @@ In CCC, the revision of this section was good overall, with some minor and easy-
 
 We also observed issues arising from revising one paragraph at a time without context.
 For instance, in PhenoPLIER, one of the first paragraphs mentions the linear models used by S-PrediXcan and S-MultiXcan without providing any equations or details.
-These were presented in the following paragraphs, but since the model had not yet encountered that information, it opted to add those equations immediately (in the correct Manubot/Markdown format).
+These were presented in the following paragraphs, but since the reader had not yet encountered that information, they might expect those equations to be added immediately (in the correct Manubot/Markdown format).
```

Comment:
```suggestion
These were presented in the following paragraphs, but since the model had not yet encountered that information, it opted to add those equations immediately (in the correct Manubot/Markdown format).
```